instrumentation/opentelemetry-instrumentation-openai/src/opentelemetry/instrumentation/openai_v2/version.py

```diff
@@ -0,0 +1,15 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+__version__ = "2.0.0.dev"
```

Comment:
We are going to be starting the package version at 2.X?

Reply:
Yeah, since this will change with semconvs I would have started at < 1.0.0

Reply:
based on @lmolkova previous comment, yes

Reply:
@lmolkova 

Please confirm reasoning for above versioning.